### PR TITLE
docs: Add guide how to use lpvs-x.y.z.jar in other project

### PR DESCRIPTION
**Pull request by tdrozdovsky**
# Pull Request

## Description

External projects can use the package, so instruction have been created on how to do this

## Type of change

- [x] Documentation update

## Testing

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] My code meets the required code coverage for lines (90% and above)
- [x] My code meets the required code coverage for branches (80% and above)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by userai-dev — APPROVED** [review]
LGTM

**Review by o-konoval — APPROVED** [review]
+1

**Review by t-naumenko — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -276,37 +276,102 @@ To build LPVS from source code and run it in the default service mode, follow th
 
 LPVS is now built and running. You can create a new pull request or update an existing one with commits, and LPVS will automatically start scanning and provide comments about the licenses found in the project.
 
-#### 4.b Single scan mode
+#### 4.2 Single scan mode
 
 Alternatively, you can perform a one-time scan on a specific pull request using the single scan mode. Follow these steps:
 
-4.b.1. Begin by running the installation and navigating to the target directory, similar to the process in service mode (refer to steps 4.a.1 and 4.a.2):
+4.2.1 Begin by running the installation and navigating to the target directory, similar to the process in service mode (refer to steps 4.2.1 and 4.2.2):
```
Comment (o-kopysov) [inline README.md:283]: Must be 4.1.1 and 4.1.2

File: README.md [hunk 2]
```diff
@@ -276,37 +276,102 @@ To build LPVS from source code and run it in the default service mode, follow th
...
 
-4.b.4. Available option to generate an HTML report and save it in a specified folder. Replace `/path/to/your/folder` with the full path to the folder where you want to save the HTML report, and `your_report_filename.html` with the desired filename for the report.
+4.2.4 Available option to generate an HTML report and save it in a specified folder. Replace `/path/to/your/folder` with the full path to the folder where you want to save the HTML report, and `your_report_filename.html` with the desired filename for the report.
    ```bash
    java -jar -Dgithub.token=<my-token> lpvs-*.jar --github.pull.request=<PR URL> --build.html.report=</path/to/your/folder/your_report_filename.html>
    ```
 
 These steps streamline the process, allowing you to run a scan on a single pull request without the need for a preconfigured database.
 
+#### 4.3 Use of lpvs-x.y.z.jar in your project
```
Comment (o-kopysov) [inline README.md:310]: 4.3 Use of LPVS JAR `lpvs-x.y.z.jar` in your project

File: README.md [hunk 3]
```diff
@@ -276,37 +276,102 @@ To build LPVS from source code and run it in the default service mode, follow th
...
+
+4.3.1 Authenticating with a personal access tocken
```
Comment (o-kopysov) [inline README.md:312]: Token

File: README.md [hunk 4]
```diff
@@ -276,37 +276,102 @@ To build LPVS from source code and run it in the default service mode, follow th
...
+
+You can authenticate to GitHub Packages with Apache Maven by editing your `~/.m2/settings.xml` file to include your personal access token
+> Create a token with minimally sufficient rights:
+>  - Fine-grained tockens **(recommended)**
```
Comment (o-kopysov) [inline README.md:316]: Token

File: README.md [hunk 5]
```diff
@@ -276,37 +276,102 @@ To build LPVS from source code and run it in the default service mode, follow th
...
 
+#### 4.3 Use of lpvs-x.y.z.jar in your project
+
+4.3.1 Authenticating with a personal access tocken
+
+You can authenticate to GitHub Packages with Apache Maven by editing your `~/.m2/settings.xml` file to include your personal access token
+> Create a token with minimally sufficient rights:
+>  - Fine-grained tockens **(recommended)**  
+      _Only select repositories -> Permissions -> Repository permissions -> Metadata -> Read-only_
+>  - Tockens (classic)
```
Comment (o-kopysov) [inline README.md:318]: Tokens

File: README.md [hunk 6]
```diff
@@ -276,37 +276,102 @@ To build LPVS from source code and run it in the default service mode, follow th
...
+  <servers>
+    <server>
+      <id>github</id>
+      <username>USERNAME</username>
+      <password>TOKEN</password>
+    </server>
+  </servers>
+</settings>
+```
+> if your `settings.xml` file is not located in `~/.m2/settings.xml`, then need to add the `-s path/to/file/settings.xml` option to `mvn` command
```
Comment (o-kopysov) [inline README.md:358]: then you need

**Review by o-kopysov — COMMENTED** [review]
Please check my comments

File: README.md (hunk 1 above)
Comment (tdrozdovsky) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (tdrozdovsky) [reply README.md]: Done!

File: README.md (hunk 3 above)
Comment (tdrozdovsky) [reply README.md]: Done!

File: README.md (hunk 4 above)
Comment (tdrozdovsky) [reply README.md]: Done!

File: README.md (hunk 5 above)
Comment (tdrozdovsky) [reply README.md]: Done!

File: README.md (hunk 6 above)
Comment (tdrozdovsky) [reply README.md]: Done!

**Review by o-kopysov — APPROVED** [review]
LGTM

**Pull request merged**